core/droidkaigiui/src/commonMain/kotlin/io/github/droidkaigi/confsched/droidkaigiui/component/AnimatedTextTopAppBar.kt

```diff
@@ -59,6 +60,7 @@ fun AnimatedTextTopAppBar(
     }
     val density = LocalDensity.current.density
     var navigationIconWidthDp by remember { mutableStateOf(0f) }
+    val currentTextColor = if (transitionFraction > 0f) scrolledTextColor else textColor
```

Comment:
Without this processing, text becomes indistinguishable from the background color when the header background is displayed.
Furthermore, if the background during scrolling is also made transparent, it blends with the background color and the text becomes invisible.
Therefore, the background color during scrolling should not be made transparent like other screens; instead, it was necessary to allow the text color to be specified to match the displayed background color.

https://github.com/user-attachments/assets/ef33a4f4-657b-4b08-ac87-15c9d2be3f5b